src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -282,7 +282,11 @@ public function process()
                 $bMsg .= 'ActionId: ' . $this->_lastActionId . "\r\n" . $aMsg;
                 $event = $this->_messageToEvent($bMsg);
                 $response = $this->findResponse($event);
-                $response->addEvent($event);
+                if ($response === false) {
```

Comment:
Hi :) Could you give a use case? When/Why is this happening? What event(s) and asterisk version(s)? Could you paste the raw events (and actions) that cause this?